_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TEAM 1st wins Student Robotics Competition 2022!
+image: image
+image_alt: Alt text for the image
+description: SR2022 came to an exciting conclusion with a victory for HRS
+---
+
+That’s it for Student Robotics 2022! HRS took the crown in an intense
+showdown between CCR, BRK, and QMC. It was fantastic to finally
+see robots in-person again after two years of being virtual. The ingenuity and
+engineering displayed by all our teams was incredible and we're delighted with
+the performances we saw over the weekend.
+
+## The Game: This Way Up
```

Comment:
Would be great to do a pass through this section to swap "team" for "robot" as we did in the booklet?